pkg/build/scanner.go

```diff
@@ -63,8 +63,9 @@ func Scanner(manifest model.Manifest, githubToken string) error {
 		if err != nil {
 			if err, ok := err.(net.Error); ok && err.Timeout() {
 				fmt.Println("Scanner request timed out. Need to run request again.")
+			} else {
+				break
```

Comment:
we never return the error now as far as I can tell

Reply:
err should be set and returned in the check immediately after the `for` loop

Reply:
That said, I can simply return the err here with the same behavior.